docusaurus/docs/introduction/backend.md

```diff
@@ -83,20 +79,29 @@ Examples of use cases for implementing resources:
 - Extend Grafana's HTTP API with custom resources, methods and actions.
 - Use [chunked transfer encoding](https://en.wikipedia.org/wiki/Chunked_transfer_encoding) to return large data responses in chunks or to enable certain streaming capabilities.
 
-#### Health checks
+### Health checks
 
 The health checks capability allows a backend plugin to return the status of the plugin. For data source backend plugins, the health check is automatically called when a user edits a data source and selects _Save & Test_ in the UI.
 
 A plugin's health check endpoint is exposed in the Grafana HTTP API and allows external systems to continuously poll the plugin's health to make sure that it's running and working as expected.
 
-#### Collect metrics
+### Collect metrics
 
 A backend plugin can collect and return runtime, process, and custom metrics using the text-based Prometheus [exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/). If you're using the [Grafana Plugin SDK for Go](./grafana-plugin-sdk-for-go.md) to implement your backend plugin, then the [Prometheus instrumentation library for Go applications](https://github.com/prometheus/client_golang) is built-in. This SDK gives you Go runtime metrics and process metrics out of the box. You can use the [Prometheus instrumentation library](https://github.com/prometheus/client_golang) to add custom metrics to instrument your backend plugin.
 
 The Grafana HTTP API offers an endpoint (`/api/plugins/<plugin id>/metrics`) that allows you to configure a Prometheus instance to scrape the metrics.
 
-#### Streaming
+### Streaming
+
+The streaming capability allows a backend plugin to handle data source queries that are streaming. For more information, refer to an example for a [streaming data source plugin](https://github.com/grafana/grafana-plugin-examples/tree/main/examples/datasource-streaming-backend-websocket).
+
+## Data communication model
+
+Grafana uses a simplified model where all necessary information (configuration) is provided in each request to a backend plugin, allowing the plugin to fulfill the request and return a response. 
+
+## Caching and connection pooling
```

Comment:
Been reading this several times now and keep coming back to whether this belongs in this introduction page. I'm leaning towards no and instead create a separate page/topic where we can go into more details and include actual code examples with instructions how to properly make use of the instance management and implement connection pooling.

My suggestion is to skip this section for now. WDYT?

Reply:
My preference would be to include the caching section even if it isn't as great as it could be, and then create a better caching topic down the road with specific instructions and code examples. Unless you think that providing incomplete info now could potentially confuse or frustrate users more than including it, then let's leave it out.

Reply:
Sorry for slow response. Sure we can include it, but think we need to change some of it. Let me see if I can come up with something. Main thing being that a plugin author need to opt in to the instance management to make use of it.

Reply:
@marefr - I have changed to this for now:

Grafana uses a communication model where you can opt in to instance management to simplify the development process. If you do, then all necessary information (configuration) is provided in each request to a backend plugin, allowing the plugin to fulfill the request and return a response. This model simplifies for plugin authors not having to keep track of or request additional state to fulfill a request.

It could benefit from explaining how someone opts in to instance management. Let me know if you'd like to see it changed or, after further consideration, the section removed.

Reply:
i think this is handled by the two paragraphs you've written below. unresolve if that is not the case.